GcmEndpoints/README.md

```diff
@@ -39,23 +41,7 @@ To ensure that your backend started successfully, navigate to [http://localhost:
 
 ## 2.1. Obtaining Google Cloud Messaging API key
 
-Before testing the application, you will need to obtain a Google Cloud Messaging API key. Open the `<backend>/src/main/webapp/WEB-INF/appengine-web.xml` file (it should have been opened by default when you generated the backend), and navigate to this link: [https://console.developers.google.com/flows/enableapi?apiid=googlecloudmessaging&keyType=SERVER_SIDE&r=0.0.0.0/0](https://console.developers.google.com/flows/enableapi?apiid=googlecloudmessaging&keyType=SERVER_SIDE&r=0.0.0.0/0).
-
-![Project creation in Google Developers Console](/doc/img/dev-console-gcm-register.png)
-
-Choose a "Create a new project" option to create a new [Google Developers Console](https://console.developers.google.com) project (or choose an existing project, if you have one already), and click "Continue".
-
-Read the Terms of Service and assuming that you agree, check the checkbox to indicate that you have read them and click "Accept".
-
-Once you select or create a project, it will have "Google Cloud Messaging for Android" API enabled automatically. In the following configuration dialog, you can use the supplied "0.0.0.0/0" IP address for testing purposes.
-
-![Creating a new server key in Google Developers Console](/doc/img/dev-console-server-key.png)
-
-Click "Create" to get the API key for server applications generated for you.
-
-![Server applications API key in Google Developers Console](/doc/img/dev-console-key-view.png)
-
-Copy the generated [API key](http://developer.android.com/google/gcm/gcm.html#apikey) (in a red rectangle, starts with `AIza...`) back into `appengine-web.xml` file, replacing
+Before testing the application, you will need a Google Cloud Messaging API key. Follow the [instructions](https://developers.google.com/cloud-messaging/android/client) to create a `google-services.json` configuration file and API key. Open the `<backend>/src/main/webapp/WEB-INF/appengine-web.xml` file (it should have been opened by default when you generated the backend), and copy the generated API key into `appengine-web.xml` file, replacing
```

Comment:
"To test the application, you need a Google Cloud Messaging API key."

(In general, tech writing lives in the present tense.)